lbrynet/daemon/Publisher.py

```diff
@@ -56,28 +56,22 @@ def create_and_publish_stream(self, name, bid, claim_dict, file_path, claim_addr
                 log.info("Removed old stream for claim update: %s", lbry_file.stream_hash)
 
         yield self.storage.save_content_claim(
-            self.lbry_file.stream_hash, "%s:%i" % (claim_out['txid'], claim_out['nout'])
+            self.lbry_file.stream_hash, tx.outputs[0].id
```

Comment:
It's confusing that this is the outpoint string - I thought this was the claim id.